### PR TITLE
allow manual initialization of the directionalnavigation - should hel…

### PR DESCRIPTION
**Pull request by staxmanade**
…p with #21 

I'm using [jspm](http://jspm.io) to load the library and also need a way to initialize it.

# How To Install:

```
jspm install TVHelpers=github:Microsoft/TVHelpers
```

# Import (to use the code)
```
import { DirectionalNavigation } from 'TVHelpers/tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0';

// force initialization
DirectionalNavigation.init();
```

Thoughts?

### CODE REVIEW

File: tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0.js [hunk 1]
```diff
@@ -819,7 +819,12 @@
         }
     });
 
-    document.addEventListener("DOMContentLoaded", function () {
+    var _initRun = false;
+    var init = function () {
```
Comment (payzer) [inline tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0.js]: To follow the convention, could you name this function with an underscore? "_init".

File: tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0.js [hunk 2]
```diff
@@ -957,6 +966,7 @@
     };
     // Publish to WinJS namespace
     var toPublish = {
+        init: init,
```
Comment (payzer) [inline tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0.js]: Then, down here change it to init: _init,

**Comment by staxmanade** [comment]
@payzer thanks for the feedback. 👍

**Comment by payzer** [comment]
Thank you. The change looks good.

**Comment by staxmanade** [comment]
YAY! Thanks for the quick turn-around.

**Pull request merged**